resources/lang/id/beatmaps.php

```diff
@@ -0,0 +1,282 @@
+        'message_placeholder_deleted_beatmap' => 'Tingkat kesulitan ini telah dihapus sehingga diskusi lebih lanjut tidak lagi diperkenankan.',
+        'message_type_select' => 'Pilih Jenis Komentar',
+        'reply_notice' => 'Tekan enter untuk membalas',
+        'reply_placeholder' => 'Ketik balasan Anda di sini',
+        'require-login' => 'Silakan masuk untuk posting atau membalas',
+        'resolved' => 'Terselesaikan',
+        'restore' => 'pulihkan',
+        'title' => 'Diskusi',
+
+        'collapse' => [
+            'all-collapse' => 'Ciutkan semua',
+            'all-expand' => 'Lebarkan semua',
+        ],
+
+        'empty' => [
+            'empty' => 'Belum ada diskusi!',
+            'hidden' => 'Tidak ada diskusi yang cocok dengan filter yang dipilih.',
+        ],
+
+        'message_hint' => [
+            'in_general' => 'Topik-topik diskusi ini berlaku untuk keseluruhan mapset secara umum. Untuk membuka topik diskusi baru, mulai pesan Anda dengan keterangan waktu (cth : 00:12:345).',
+            'in_timeline' => 'Topik-topik diskusi ini berlaku untuk masing-masing tingkat kesulitan secara spesifik. Untuk memulai topik diskusi baru, salin keterangan waktu dari editor disertai dengan komentar Anda (satu topik per keterangan waktu).',
+        ],
+
+        'message_type' => [
+            'disqualify' => 'Diskualifikasi',
+            'hype' => 'Hype!',
+            'mapper_note' => 'Catatan',
+            'nomination_reset' => 'Hilangkan Status Nominasi',
+            'praise' => 'Pujian',
+            'problem' => 'Masalah',
+            'suggestion' => 'Saran',
+        ],
+
+        'mode' => [
+            'events' => 'Peristiwa',
+            'general' => 'Umum :scope',
+            'timeline' => 'Linimasa',
+            'scopes' => [
+                'general' => 'Tingkat kesulitan ini',
+                'generalAll' => 'Semua tingkat kesulitan',
+            ],
+        ],
+
+        'new' => [
+            'timestamp' => 'Keterangan Waktu',
+            'timestamp_missing' => 'Salin (ctrl+c) keterangan waktu yang spesifik dari editor dan tempelkan (ctrl+v) pada boks yang tersedia untuk menambahkan keterangan waktu!',
+            'title' => 'Diskusi Baru',
+        ],
+
+        'show' => [
+            'title' => ':title dibuat oleh :mapper',
+        ],
+
+        'sort' => [
+            '_' => 'Urut berdasarkan:',
+            'created_at' => 'waktu pembuatan',
+            'timeline' => 'linimasa',
+            'updated_at' => 'pembaruan terakhir',
+        ],
+
+        'stats' => [
+            'deleted' => 'Dihapus',
+            'mapper_notes' => 'Catatan',
+            'mine' => 'Milik Saya',
+            'pending' => 'Menunggu',
+            'praises' => 'Pujian',
+            'resolved' => 'Terselesaikan',
+            'total' => 'Semua',
+        ],
+
+        'status-messages' => [
+            'approved' => 'Beatmap ini telah di-approve pada :date!',
```

Comment:
di-approve or disetujui?

Reply:
Thank god you remind me to capitalize the approve.
I won't change it to "disetujui", because it's a term.